content/2015.12.what-can-kinto-do.rst

```diff
@@ -23,14 +23,125 @@ Here is an attempt to fix this.
 Kinto is not a framework
 ########################
 
-First and foremost, Kinto is not a framework. Kinto is meant to replace the
+First and foremost, Kinto is not a framework. It doesn't try to provide a way
+to structure your projects for you. It is meant to replace the
 storage layer of your application, or more accurately, the synchronisation
 abilities of your applications.
 
-Said differentely, you store your data locally for your application and Kinto
-will make this data available to different devices. That means that you can use
-Kinto with React, Angular, JQuery (in the JavaScript world) or any framework in
-any language. Kinto abstracts the synchronisation of the application's data.
+Said differentely, you store your data locally for your application and **Kinto
+will make this data available to different devices (or users)**.
+
+That means that you can use Kinto with React, Angular, JQuery (in the
+JavaScript world) or any framework (or no framework at all) in any language.
+**Kinto abstracts away the synchronisation of the application's data**.
+
+Hmm, and what about my use-case?
+################################
+
+The idea here is mainly to show you what is already possible with very little
+efforts using the tooling we have put together. You can leverage the
+`Kinto.js <https://kintojs.readthedocs.org>`_ client, as it allows you to work
+with Kinto really easily, but ultimately you don't need it. It's just here to
+make it really easy for you to integrate with the HTTP APIs exposed by Kinto.
```

Comment:
I disagree with the idea that you don't need it. Kinto.js is pretty important for offline-first access.